riddlc/src/test/scala/com/reactific/riddl/example.riddl

```diff
@@ -0,0 +1,29 @@
+domain Example is {
+	context ExampleContext is {
+		command DoFoo is {foo: Foo}
+		type Foo is {
+			bar: String
+		}
+		entity FooEntity is {
+			record FooExampleState is {
+	        foo: Foo
+	    }
+	    state FooExample of FooExampleState is {
+	        handler HandleFoo is {
+	          on command DooFoo {
+		          example Success {
+		            then set FooExamplexxxx.garbage to !Foo(garbage = "garbage")
+		          }
+	          }
+	        }
+	    }
```

Comment:
Spacing and indent makes the above entity difficult to comprehend because the state is less indented than the entity.